css/layout.css

```diff
@@ -58,4 +68,15 @@
   .justify-center-mobile {
     justify-content: flex-start;
   }
+
+  .modal-content {
+    position: relative;
+    top: -25%;
+    background-color: #fefefe;
+    margin: 25% auto;
+    padding: 20px;
+    width: 60%;
+    height: 70%;
```

Comment:
What happens if you remove this height? You shouldn't set a height on an element unless it is _absolutely_ necessary.